web/src/hashchange.ts

```diff
@@ -153,7 +134,6 @@ function show_home_view(): void {
             break;
         }
         case "inbox": {
-            maybe_hide_recent_view();
```

Comment:
I guess this matches the `show_all_message_view` pattern. In general, this is a place where a good commit message would explain the benefits of the change, not just what we're doing.

Reply:
Added this to commit message

> This will help as we will add other views to the app to keep the logic to hide other views in the show function of the view to show. So, the logic calling show for a view doesn't have to iterate through hiding all the other views first.

Reply:
The trade-off here will be that we need to make sure each individual view's code has logic to hide each other possible view type. But maybe that's hard to avoid, since at least in some cases, we seem to care what the previous view was, and/or want to hide the other view at the right moment, say after fetching some data.